### PR TITLE
Add Void65h

### PR DESCRIPTION
**Pull request by EasonQian1**
## Description
Add Void65h

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/eason/void65h/info.json
Comment (waffle87) [inline keyboards/eason/void65h/info.json:1]: Please run `qmk format-json` on this file.

Comment (waffle87) [inline keyboards/eason/void65h/info.json:1]: This file should also be renamed to `keyboard.json`.

File: keyboards/eason/void65h/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______,_______,_______
+    ),
+
+    [3] = LAYOUT(
+        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______,_______,_______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,_______,_______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,_______,_______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,_______,_______,
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______,_______,_______
+    ),
```
Comment (waffle87) [inline keyboards/eason/void65h/keymaps/via/keymap.c:37]:
```suggestion
    )
```

File: keyboards/eason/void65h/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# void65h
+
+![meow65](https://i.imgur.com/3Z3jNAS.jpg)
+
+* A customizable hotswap 65% keyboard.
```
Comment (waffle87) [inline keyboards/eason/void65h/readme.md:5]:
```suggestion
A customizable hotswap 65% keyboard.
```

File: keyboards/eason/void65h/rules.mk
Comment (waffle87) [inline keyboards/eason/void65h/rules.mk:1]: This file should be removed.

**Review by waffle87 — CHANGES_REQUESTED** [review]
Keyboard doesn't compile:
```
error: macro "LAYOUT" passed 74 arguments, but takes just 67
```

File: keyboards/eason/void65h/keyboard.json [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                  {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
+                  {"matrix": [4, 1], "x": 1.5, "y": 4},
+                  {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
+                  {"matrix": [4, 6], "x": 4, "y": 4, "w": 7},
+                  {"matrix": [4, 11], "x": 11, "y": 4, "w": 1.5},
+                  {"matrix": [4, 12], "x": 13, "y": 4},
+                  {"matrix": [4, 13], "x": 14, "y": 4},
+                  {"matrix": [4, 14], "x": 15, "y": 4}
+             ]
+          }
```
Comment (dunk2k) [inline keyboards/eason/void65h/keyboard.json:102]:
```suggestion
        },
        "LAYOUT_65_ansi_blocker_tsangan": {
              "layout": [
                  {"matrix": [0, 0], "x": 0, "y": 0},
                  {"matrix": [0, 1], "x": 1, "y": 0},
                  {"matrix": [0, 2], "x": 2, "y": 0},
                  {"matrix": [0, 3], "x": 3, "y": 0},
                  {"matrix": [0, 4], "x": 4, "y": 0},
                  {"matrix": [0, 5], "x": 5, "y": 0},
                  {"matrix": [0, 6], "x": 6, "y": 0},
                  {"matrix": [0, 7], "x": 7, "y": 0},
                  {"matrix": [0, 8], "x": 8, "y": 0},
                  {"matrix": [0, 9], "x": 9, "y": 0},
                  {"matrix": [0, 10], "x": 10, "y": 0},
                  {"matrix": [0, 11], "x": 11, "y": 0},
                  {"matrix": [0, 12], "x": 12, "y": 0},
                  {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                  {"matrix": [0, 15], "x": 15, "y": 0},
                  {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                  {"matrix": [1, 1], "x": 1.5, "y": 1},
                  {"matrix": [1, 2], "x": 2.5, "y": 1},
                  {"matrix": [1, 3], "x": 3.5, "y": 1},
                  {"matrix": [1, 4], "x": 4.5, "y": 1},
                  {"matrix": [1, 5], "x": 5.5, "y": 1},
                  {"matrix": [1, 6], "x": 6.5, "y": 1},
                  {"matrix": [1, 7], "x": 7.5, "y": 1},
                  {"matrix": [1, 8], "x": 8.5, "y": 1},
                  {"matrix": [1, 9], "x": 9.5, "y": 1},
                  {"matrix": [1, 10], "x": 10.5, "y": 1},
                  {"matrix": [1, 11], "x": 11.5, "y": 1},
                  {"matrix": [1, 12], "x": 12.5, "y": 1},
                  {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                  {"matrix": [1, 14], "x": 15, "y": 1},
                  {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                  {"matrix": [2, 1], "x": 1.75, "y": 2},
                  {"matrix": [2, 2], "x": 2.75, "y": 2},
                  {"matrix": [2, 3], "x": 3.75, "y": 2},
                  {"matrix": [2, 4], "x": 4.75, "y": 2},
                  {"matrix": [2, 5], "x": 5.75, "y": 2},
                  {"matrix": [2, 6], "x": 6.75, "y": 2},
                  {"matrix": [2, 7], "x": 7.75, "y": 2},
                  {"matrix": [2, 8], "x": 8.75, "y": 2},
                  {"matrix": [2, 9], "x": 9.75, "y": 2},
                  {"matrix": [2, 10], "x": 10.75, "y": 2},
                  {"matrix": [2, 11], "x": 11.75, "y": 2},
                  {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                  {"matrix": [2, 14], "x": 15, "y": 2},
                  {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                  {"matrix": [3, 2], "x": 2.25, "y": 3},
                  {"matrix": [3, 3], "x": 3.25, "y": 3},
                  {"matrix": [3, 4], "x": 4.25, "y": 3},
                  {"matrix": [3, 5], "x": 5.25, "y": 3},
                  {"matrix": [3, 6], "x": 6.25, "y": 3},
                  {"matrix": [3, 7], "x": 7.25, "y": 3},
                  {"matrix": [3, 8], "x": 8.25, "y": 3},
                  {"matrix": [3, 9], "x": 9.25, "y": 3},
                  {"matrix": [3, 10], "x": 10.25, "y": 3},
                  {"matrix": [3, 11], "x": 11.25, "y": 3},
                  {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                  {"matrix": [3, 13], "x": 14, "y": 3},
                  {"matrix": [3, 14], "x": 15, "y": 3},
                  {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                  {"matrix": [4, 1], "x": 1.5, "y": 4},
                  {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                  {"matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                  {"matrix": [4, 11], "x": 11, "y": 4, "w": 1.5},
                  {"matrix": [4, 12], "x": 13, "y": 4},
                  {"matrix": [4, 13], "x": 14, "y": 4},
                  {"matrix": [4, 14], "x": 15, "y": 4}
             ]
        }
```

File: keyboards/eason/void65h/keyboard.json [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    },
+    "matrix_pins": {
+        "rows": ["A15", "B3", "B4", "B5", "B6"],
+        "cols": ["A1", "A2", "A3", "A4", "A5", "A6", "A7", "B0", "B1", "B10", "B11", "B12", "B13", "B14", "B15", "A8"]
+    },
+    "diode_direction": "ROW2COL",
+    "processor": "STM32F103",
+    "bootloader": "uf2boot",
+    "layouts": {
+          "LAYOUT": {
```
Comment (dunk2k) [inline keyboards/eason/void65h/keyboard.json:32]:
```suggestion
    "layout_aliases": {
        "LAYOUT_all": "LAYOUT_65_ansi_blocker_tsangan_split_bs"
    },
    "community_layouts": ["65_ansi_blocker_tsangan", "65_ansi_blocker_tsangan_split_bs"]
    "layouts": {
          "LAYOUT_65_ansi_blocker_tsangan_split_bs": {
```

File: keyboards/eason/void65h/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  KC_PGDN,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT, KC_UP,   MO(1),
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  MO(1),   KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT(
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______
```
Comment (dunk2k) [inline keyboards/eason/void65h/keymaps/default/keymap.c:19]:
```suggestion
    [0] = LAYOUT_all(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_TILD, KC_DEL,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT, KC_UP,   MO(1),
        KC_LCTL, KC_LGUI, KC_LALT,                   KC_SPC,                                               MO(1),   KC_LEFT, KC_DOWN, KC_RGHT
    ),
    [1] = LAYOUT_all(
        KC_TILD, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______,          _______, _______, _______, _______, QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,                   _______,                                              _______, _______, _______, _______
```

File: keyboards/eason/void65h/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,  KC_PGDN,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT, KC_UP,   MO(1),
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  MO(1),   KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT(
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______
```
Comment (dunk2k) [inline keyboards/eason/void65h/keymaps/via/keymap.c:19]:
```suggestion
    [0] = LAYOUT_all(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_TILD, KC_DEL,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT, KC_UP,   MO(1),
        KC_LCTL, KC_LGUI, KC_LALT,                   KC_SPC,                                               MO(1),   KC_LEFT, KC_DOWN, KC_RGHT
    ),
    [1] = LAYOUT_all(
        KC_TILD, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______,          _______, _______, _______, _______, QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______,                   _______,                                              _______, _______, _______, _______
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
- Community Layout support
- add `LAYOUT_all` alias to conform to PR guidelines
- format keymaps
  - add bootmagic keycode

File: keyboards/eason/void65h/keyboard.json [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+        "force_nkro": true
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": false
```
Comment (waffle87) [inline keyboards/eason/void65h/keyboard.json:19]:
```suggestion
        "nkro": true
```

File: keyboards/eason/void65h/readme.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# void65h
+
+![meow65](https://i.imgur.com/3Z3jNAS.jpg)
```
Comment (waffle87) [inline keyboards/eason/void65h/readme.md:3]: Unless this is intentional?
```suggestion
![void65h](https://i.imgur.com/3Z3jNAS.jpg)
```

**Review by waffle87 — APPROVED** [review]
Otherwise looks good to me.

**Review by drashna — APPROVED** [review]

File: keyboards/eason/void65h/readme.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# void65h
+
+![void65h](https://i.imgur.com/3Z3jNAS.jpg)
+
+A customizable hotswap 65% keyboard.
+
+* Keyboard Maintainer: [EASON](https://github.com/EasonQian1)
+* Hardware Supported: void65h
+* Hardware Availability: [vertex-kb](https://github.com/Vertex-kb)
```
Comment (drashna) [inline keyboards/eason/void65h/readme.md:9]: Is there an actual public repo for this, or otherwise a link for a place to purchase?

Comment (EasonQian1) [reply keyboards/eason/void65h/readme.md]: Corrected

File: keyboards/eason/void65h/readme.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+    make eason/void65h:default
+
+Flashing example for this keyboard:
+
+    make eason/void65h:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader Enter the bootloader in 3 ways:
```
Comment (zvecr) [inline keyboards/eason/void65h/readme.md:21]:
```suggestion
## Bootloader

Enter the bootloader in 3 ways:

```

**Pull request merged**